components/services/bat_ledger/BUILD.gn

```diff
@@ -1,7 +1,8 @@
 static_library("lib") {
   visibility = [
     "//brave/utility:*",
-    "//brave/test:*"
+    "//brave/test:*",
+    "//chrome/utility:*"
```

Comment:
Without that change it is not buildable on Android. Is there any problem with this dependency?

Reply:
I don't have problem with it, just want to understand it. @bridiver @yrliou does this looks good to you to be merged into core? or we want to fix it so that it will work without adding it

Reply:
@samartnik that doesn't make sense to me because there shouldn't be a direct dep on this in chrome/utility and this isn't a dependency, it's for visibility

Reply:
what is the exact build error you're seeing without it?

Reply:
In our current (non brave-core) implementation we have `bat_ledger:lib` dependency. It requires this visibility. So we need this change temporarily, till we move Android on brave-core. Not sure how to workaround it.

Reply:
same as above, use //brave/utility as the dep instead to avoid changing the visibility list. These are here specifically to prevent exactly the thing you are doing :)

Reply:
Removed